packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -21,6 +21,7 @@ import {
 import { Link } from 'react-router-dom';
 import { localize, Localize } from '@deriv/translations';
 import { getLegalEntityName, isDesktop, isMobile, routes, toMoment, PlatformContext } from '@deriv/shared';
+import { getEmploymentStatusList } from 'Sections/Assessment/FinancialAssessment/financial-information-list.js';
```

Comment:
Since we are using getEmploymentStatusList in Sections and Components, can we move this function to helpers folder?

Reply:
.js extension is not needed. It will be resolved automatically.